engine/menu/debug_menu.asm

```diff
@@ -82,26 +82,151 @@ DebugMenuOptionSubGames::
 DebugMenuOptionMonsterTest::
 	ld hl, wPokedexOwned
 	ld de, wPokedexSeen
-	ld b, $1F
-	ld a, $FF
+	ld b, 31
```

Comment:
When the meaning of a hard-coded number isn't obvious, it should just stay as hex. In this case, `$1F` / `31` is `NUM_POKEMON / 8`.